pkg/register/http.go

```diff
@@ -10,19 +10,25 @@ import (
 	http_transport "github.com/go-kit/kit/transport/http"
 )
 
+// RegistContext type
+type RegistContext int
+
+// Regular expressions
 const (
-	// RegExpRealmName is a regular expression for realm names
+	RegistContextRecaptcha RegistContext = iota
```

Comment:
I don't understand what is it and where we use it ?

Reply:
Removed